core/node/eigenda_proxy/Cargo.toml

```diff
@@ -14,3 +14,9 @@ anyhow.workspace = true
 axum.workspace = true
 tokio.workspace = true
 tracing.workspace = true
+prost = "0.13.1"
```

Comment:
`prost` is available as a `workspace` dependency.

Reply:
the prost version used workspace wide is older than the one required for tonic. I added a comment explaining that